### PR TITLE
[Enhancement] (nereids)implement showBackendsCommand in nereids

### LINKED ISSUE #42762: [Enhancement] (nereids)implement showBackendsCommand in nereids
showBackends command is supported in legacy planner but not in nereids(new doris fe planner). you need implement the following showBackends command in nereids:
unsupportedshowBackendsStatement
``` SHOW BACKENDS                                                                 ```

### PR DESCRIPTION
**Pull request by Vallishp**
### What problem does this PR solve?

Support Show Backends in nereids

Issue Number: close #42762 

Related PR: #xxx

Problem Summary:

### Check List (For Committer)

- Test 

    - [ ] Regression test yes
    - [ ] Unit Test
    - [ ] Manual test (add detailed scripts or steps below)
    - [ ] No need to test or manual test. Explain why:
        - [ ] This is a refactor/code format and no logic has been changed.
        - [ ] Previous test can cover this change.
        - [ ] No colde files have been changed.
        - [ ] Other reason 

Already testcase present test_backend.groovy
 RemoteUsedCapacity:0.000 , BackendId:10002, CpuCores:8, LastStartTime:2024-11-01 17:45:25, Tag:{"location" : "default"}]]
2024-11-01 17:49:17.647 INFO [non-concurrent-thread-1] (Suite.groovy:384) - Execute sql: ALTER SYSTEM DECOMMISSION BACKEND "10002" 
2024-11-01 17:49:17.666 INFO [non-concurrent-thread-1] (test_backend.groovy:60) - ALTER SYSTEM DECOMMISSION BACKEND [[0]]
2024-11-01 17:49:17.667 INFO [non-concurrent-thread-1] (Suite.groovy:451) - Execute sql: SHOW BACKENDS;
2024-11-01 17:49:17.694 INFO [non-concurrent-thread-1] (Suite.groovy:384) - Execute sql: CANCEL DECOMMISSION BACKEND "10002" 
2024-11-01 17:49:17.707 INFO [non-concurrent-thread-1] (test_backend.groovy:71) - CANCEL DECOMMISSION BACKEND [[0]]
2024-11-01 17:49:17.708 INFO [non-concurrent-thread-1] (Suite.groovy:451) - Execute sql: SHOW BACKENDS;
2024-11-01 17:49:17.718 INFO [non-concurrent-thread-1] (Suite.groovy:451) - Execute sql: show frontends
2024-11-01 17:49:17.805 INFO [non-concurrent-thread-1] (DebugPoint.groovy:162) - disable debug point SystemHandler.decommission_no_check_replica_num for FE 172.20.80.1:8030
2024-11-01 17:49:17.819 INFO [non-concurrent-thread-1] (Http.groovy:85) - http post url=http://172.20.80.1:8030/api/debug_point/remove/SystemHandler.decommission_no_check_replica_num, data=null, isJson=true, response code=200, text={"msg":"success","code":0,"data":null,"count":0}
2024-11-01 17:49:17.820 INFO [non-concurrent-thread-1] (Suite.groovy:384) - Execute sql: admin set frontend config("drop_backend_after_decommission" = "true")
2024-11-01 17:49:17.825 INFO [non-concurrent-thread-1] (ScriptContext.groovy:120) - Run test_backend in /root/doris/workspace/doris/regression-test/suites/node_p0/test_backend.groovy succeed
2024-11-01 17:49:25.546 INFO [main] (RegressionTest.groovy:410) - Success suites:
/root/doris/workspace/doris/regression-test/suites/node_p0/test_backend.groovy: group=nonConcurrent,p0, name=test_backend
2024-11-01 17:49:25.547 INFO [main] (RegressionTest.groovy:492) - All suites success.

- Behavior changed:

    - [ ] No.
    - [ ] Yes. 

- Does this need documentation?

    - [ ] No.
    - [ ] Yes. 

- Release note

    
    
    None

### Check List (For Reviewer who merge this PR)

- [ ] Confirm the release note
- [ ] Confirm test cases
- [ ] Confirm document
- [ ] Add branch pick label

### CODE REVIEW

**Comment by Vallishp** [comment]
run buildall

**Comment by Vallishp** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/ShowBackendsCommand.java [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+/**
+ * show backends command
+ */
+public class ShowBackendsCommand extends ShowCommand {
+    public static final Logger LOG = LogManager.getLogger(ShowBackendsCommand.class);
+
+    /**
+     * constructor
+     */
+
```
Comment (starocean999) [inline fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/ShowBackendsCommand.java:50]: remove empty line
```suggestion
```

**Comment by Vallishp** [comment]
run buildall

**Comment by starocean999** [comment]
please resolve conflict

**Comment by Vallishp** [comment]
run buildall

**Comment by Vallishp** [comment]
run p0

**Comment by Vallishp** [comment]
run buildall

**Comment by Vallishp** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/ShowBackendsCommand.java [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+import org.apache.logging.log4j.Logger;
+
+import java.util.Comparator;
+import java.util.List;
+
+/**
+ * show backends command
+ */
+public class ShowBackendsCommand extends ShowCommand {
+    public static final Logger LOG = LogManager.getLogger(ShowBackendsCommand.class);
```
Comment (starocean999) [inline fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/ShowBackendsCommand.java:45]:
```suggestion
```

**Review by starocean999 — APPROVED** [review]
LGTM

**Review by LiBinfeng-01 — APPROVED** [review]

**Pull request merged**